Azure/sonic-buildimage
# [build_debian] Include checksum of ASIC config files in SONiC filesystem

**- What I did**
I modified the build_debian.sh script to include a checksum of ASIC configuration files in the `/etc/sonic` directory of the built SONiC image.

**- How I did it**
I wrote a script to read through a static list of configuration files (at this stage, only those found in `sonic-swss/swssconfig/samples`) and generate an MD5 checksum of those configuration files.

**- How to verify it**
Build a new SONiC image and verify that `fsroot/etc/sonic` contains an `asic_config_checksum` file.

**- Description for the changelog**

Include checksum of ASIC config files in SONiC filesystem.

**- A picture of a cute animal (not mandatory but encouraged)**
![cute animal](https://images.unsplash.com/photo-1547660351-51b6d384015f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=200&q=80)
